src/commands/compile_raw.rs

```diff
@@ -36,10 +36,23 @@ pub fn compile(sierra_program: Value) -> Result<Value> {
     )?;
     let assembled_cairo_program = cairo_program.assemble();
 
+    // taken from cairo compiler.
+    let builtins = vec![
+            "pedersen_builtin",
+            "range_check_builtin",
+            "bitwise_builtin",
+            "ec_op_builtin",
+            "poseidon_builtin",
+        ];
```

Comment:
Why would you need it here if it is hardcoded anyways? You can just hardcode it in your project/tool/whatever you need it for

Reply:
Agreed. My thinking was that if this changes in the future, all the downstream non rust VMs won't have to watch for upstream compiler changes, only USC will have to make the changes if there are compiler changes. But, maybe that thinking is flawed. I'll remove it.